### PR TITLE
EO-435 Add diverging bar chart

### PR DESCRIPTION
**Pull request by jonambas**
Demo Code:
```js
<div style={{ width: '350px' }}>
    <DivergingBar data={[
      { key: 'abc', label: 'Label 1', value: 0.8 },
      { key: 'def', label: 'Label 2', value: -0.5 },
      { key: 'xyz', label: 'Label 7', value: -0.22 },
      { key: 'ghi', label: 'Label 3', value: -0.1 },
      { key: 'jkl', label: 'Label 4', value: 0.3 },
      { key: 'mno', label: 'Label 5', value: 0.5 },
      { key: 'pqr', label: 'Label 6', value: -0.15 }
    ]}
    selected={this.state.selected}
    onClick={this.handleClick}
    height={280}
  />
</div>
```

### CODE REVIEW

File: src/pages/signals/components/charts/divergingBar/DivergingBar.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  width: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
+};
+
+DivergingBar.defaultProps = {
+  height: 250,
+  xDomain: ['auto', 'auto'],
+  xKey: 'value',
+  yKey: 'label',
+  width: '99%',
+  negativeFill: '#DB2F2D',
```
[the remark this answers is not in the document]
Comment (jonambas) [reply src/pages/signals/components/charts/divergingBar/DivergingBar.js]: Maybe, but this is something larger I think. I want to be able to do this eventually:
```
import { blue, red } from '@sparkpost/matchbox-tokens` 
```

[the remark this answers is not in the document]
Comment (bkemper) [reply src/pages/signals/components/charts/divergingBar/DivergingBar.js]: Okay.  Nevermind.

File: src/pages/signals/components/charts/divergingBar/DivergingBar.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+  render() {
+    const { height, tooltipContent, onClick, width, xDomain, xKey, yKey } = this.props;
+    return (
+      <ResponsiveContainer height={height} width={width} className={styles.DivergingBar}>
+        <BarChart data={this.getData()} layout='vertical' barCategoryGap={2}>
+          <CartesianGrid
+            horizontal={false}
+            shapeRendering='crispEdges'
+            stroke='#d2d2d7'
```
Comment (bkemper) [inline src/pages/signals/components/charts/divergingBar/DivergingBar.js]: Is this needed?

Comment (jonambas) [reply src/pages/signals/components/charts/divergingBar/DivergingBar.js]: Recharts was setting this to #ccc which is why I used that hex value in the CSS before.

Comment (jonambas) [reply src/pages/signals/components/charts/divergingBar/DivergingBar.js]: Should I move it to the css file?

Comment (bkemper) [reply src/pages/signals/components/charts/divergingBar/DivergingBar.js]: Nope.

**Pull request merged**